pkg/agent/openflow/pipeline.go

```diff
@@ -1578,7 +1612,12 @@ func (f *featurePodConnectivity) arpNormalFlow() binding.Flow {
 		Done()
 }
 
-func (f *featureNetworkPolicy) allowRulesMetricFlows(conjunctionID uint32, ingress bool) []binding.Flow {
+func (f *featureNetworkPolicy) allowRulesMetricFlows(conjunctionID uint32, ingress, isIGMP bool, tableID uint8) []binding.Flow {
+	// Ingress for multicast is not supported currently, if the rule is ingress for multicast traffic, this function will return empty rules
+	// Egress is for IGMP only supports IGMP report, this is handled by packet. So no rule is needed here.
```

Comment:
Please rephrase this sentence. Cannot understand it.

Reply:
Probably:

Ingress rules is not enforced for multicast traffic, and egress rule enforcement for IGMP packets is handled by packetIn. So, no flows will be generated for the two cases.